package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@digicatapult/openapi-ts-template",
+  "version": "0.0.1",
+  "description": "An open api typescript nodejs/express api service template",
+  "main": "src/index.ts",
+  "scripts": {
+    "lint": "eslint .",
+    "depcheck": "depcheck",
+    "build:tsoa": "tsoa spec-and-routes",
+    "build": "tsoa spec-and-routes && tsc",
+    "start": "node build/index.js",
+    "dev": "npm run build && NODE_ENV=dev concurrently \"npx tsc --watch\" \"nodemon -q build/index.js | pino-colada\"",
```

Comment:
maybe having `npm run build` in the script is causing `depcheck` to think `concurrently` and `pino-colada` aren't in use. Does removing it make depcheck happy?